### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by jlstpaul**
This is the base  Contributor Covenant code of conduct.  All I changed was to add an email address tidestransit@gmail.com as the contact (forwards to me). If you would prefer to create some sort of contact within repository (or in another file), that's fine. Please suggest edits.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
```
Comment (e-lo) [inline CODE_OF_CONDUCT.md:42]: might want to specify which group the community leaders is? Is it the PMT? Contributors?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
```
Comment (e-lo) [inline CODE_OF_CONDUCT.md:63]: ditto on specifying *who* the community leaders actually are

**Review by e-lo — CHANGES_REQUESTED** [review]
Just think we should specify who "community leaders" are but LGTM otherwise

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (jlstpaul) [reply CODE_OF_CONDUCT.md:42]: Good point. I propose to add a new paragraph at the beginning of this section on Enforcement  Responsibilities:

The community leaders for this effort include the Leadership Team and Product Management Team as described in the Governance section of CONTRIBUTING.md.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (botanize) [reply CODE_OF_CONDUCT.md:66]: Add links to CONTRIBUTING.md, e.g., `[community](CONTRIBUTING.md#PMT) [leaders](CONTRIBUTING.md#Leadership)` or `[community leaders](CONTRIBUTING.md#project-governance)`

**Comment by botanize** [comment]
re #3

**Comment by jlstpaul** [comment]
Document is updated.  @e-lo and @j-meelah Please review and merge.

**Comment by e-lo** [comment]
> @e-lo and @j-meelah Please review and merge.

**Review by e-lo — APPROVED** [review]
LGTM

**Pull request merged**